buildSrc/src/main/groovy/io.micronaut.build.internal.r2dbc-module.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id "java-library"
+    id "io.micronaut.build.internal.r2dbc-dependencies"
+    id "io.micronaut.build.internal.module"
+}
+
+micronautBuild {
+    binaryCompatibility {
+        // Enable before Micronaut 4 release
+        enabled.set(false)
```

Comment:
why do we need this ? Are not all the modules already released? Can we remove this?

Reply:
Binary compatibility is not needed to be tracked across the major modules